modules/csv/src/csv-loader.js

```diff
@@ -163,29 +165,39 @@ function isHeaderRow(row, options) {
     return Boolean(options.csv.header);
   }
 
-  return row.every(value => typeof value === 'string');
+  return row && row.every(value => typeof value === 'string');
 }
 
-async function hasHeader(csvText, options) {
-  if (options.csv.header !== 'auto') {
```

Comment:
Where did the check for `options.csv.header` go? Header detection is shaky (e.g. all string table) and this option lets applications that know the structure of the CSV specify how it should be parsed.

Reply:
it is checked in `isHeaderRow()`